Platform Documentation.md

```diff
@@ -435,7 +435,7 @@ the credentials file.
 Set the variable `SET_ENV_VARS` using the [Custom Config Add-on] to either `false` or `true` to explicitly enable or disable
 this feature.
 
-The guides section has detailed examples about how to get the credentials in different languages ([Ruby](https://www.cloudcontrol.com/dev-center/Guides/Ruby/Add-on%20credentials), [Python](https://www.cloudcontrol.com/dev-center/Guides/Python/Add-on%20credentials), [Node.js](https://www.cloudcontrol.com/dev-center/Guides/NodeJS/Add-on%20credentials), [Java](https://www.cloudcontrol.com/dev-center/Guides/Java/Add-on%20credentials), [PHP](https://www.cloudcontrol.com/dev-center/Guides/PHP/Add-on%20credentials)).
+The guides section has detailed examples about how to get the credentials in different languages ([Ruby](https://www.cloudcontrol.com/dev-center/guides/ruby/add-on-credentials), [Python](https://www.cloudcontrol.com/dev-center/guides/python/add-on-credentials), [Node.js](https://www.cloudcontrol.com/dev-center/guides/nodejs/add-on-credentials), [Java](https://www.cloudcontrol.com/dev-center/guides/java/add-on-credentials), [PHP](https://www.cloudcontrol.com/dev-center/guides/php/add-on-credentials)).
```

Comment:
Please not the changed URLs here:

Ruby: `/dev-center/guides/ruby/add-on-credentials-2`
Python: `/dev-center/guides/python/add-on-credentials`
NodeJS: `/dev-center/guides/nodejs/add-on-credentials-1`
Java: `/dev-center/guides/java/add-on-credentials-3`
PHP: `/dev-center/guides/php/add-on-credentials-4`